Assets/C#/PlayerScripts/PlayerAbilities/Ability_LineSpawner.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ability_LineSpawner : Ability_PointSpawner {
+
```

Comment:
Might be useful to add some comments as to what the difference is between PointSpawner and LineSpawner!